datalake/src/test/java/com/sequenceiq/datalake/service/sdx/SdxServiceTest.java

```diff
@@ -729,14 +729,42 @@ void testSdxCreateMediumDutySdxNoEntitlement() throws IOException {
         final String runtime = "7.2.7";
         SdxClusterRequest sdxClusterRequest = createSdxClusterRequest(runtime, MEDIUM_DUTY_HA);
         when(sdxClusterRepository.findByAccountIdAndEnvNameAndDeletedIsNull(anyString(), anyString())).thenReturn(new ArrayList<>());
-        mockEnvironmentCall(sdxClusterRequest, CloudPlatform.AWS);
+        mockEnvironmentCall(sdxClusterRequest, CloudPlatform.AZURE);
```

Comment:
Should we also test with `CloudPlatform.GCP` to ensure that the checks are done on GCP. We should make sure that the check-removal is only for AWS and for all other platforms, we will check the entitlement.

Reply:
Ah yeah we can test with it as well.